front-end/src/screens/Settings/email.tsx

```diff
@@ -95,34 +114,60 @@ const Email = ({ className }: Props): JSX.Element => {
 	};
 
 	return (
-		<Form standalone={false}>
-			<Form.Group className={className}>
+		<Form className={className} standalone={false}>
+			<Form.Group>
 				<Form.Field width={10}>
 					<label>Email</label>
+					{editing ?
+						<input
+							value={email || ''}
+							onChange={onEmailChange}
+							placeholder='mail@example.com'
+							type='email'
+						/> :
+						<div>{email}</div>
```

Comment:
In case no email is linked, I'd add some muted text.
```suggestion
						<div className={email ? 'current-email' : 'current-email text-muted'}>
							{email ? email : 'No email linked.'}
						</div>
```

Reply:
And then adding in the CSS to have horizontally-aligned with the edit button:
``` css
.current-email {
	padding-top: 1rem;
}
```